File: StudentCredit.Application/Students/Dtos/StudentCreditDto.cs
using StudentCredit.Data.Applications.Common.Enums;
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Students.Dtos
{
    public class StudentCreditDto
    {
        public CreditType? CreditType { get; set; }
        public DateTime? ContractDate { get; set; }
        public string BankName { get; set; }
        public string EducationalQualificationName { get; set; }
        public string SpecialityName { get; set; }
        public NomenclatureDto ApplicationOneType { get; set; }
    }
}

File: WebApplication/src/modules/student/models/student-credit.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { CreditType } from "src/modules/application/common/enums/credit-type.enum";

export class StudentCreditDto {
  creditType: CreditType;
  contractDate: Date;
  bankName: string;
  educationalQualificationName: string;
  specialityName: string;
  applicationOneType: NomenclatureDto;
}